harvard-lts/Harvard-Geospatial-Library
# Repoint "report a problem" footer link to URN

**Repoint "report a problem" link to URN**
* * *

# What does this Pull Request do?
Updates "Report a problem" link in footer to point to URN (vs. old qualtrics form)

# How should this be tested?
* Rebuild HGL locally using the report-a-problem-link branch
* Inspect the "Report a Problem" link in footer to confirm that it's pointing to https://nrs.harvard.edu/urn-3:hul.ois:reportaproblem
* Click on link to confirm that it directs you to the LTS Ask page https://harvard-lts.libanswers.com/

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@enriquediaz
